modules/core/include/opencv2/core/hal/intrin_avx.hpp

```diff
@@ -1521,6 +1522,9 @@ inline v_int32x8 v_ceil(const v_float64x4& a)
 inline v_float32x8 v_cvt_f32(const v_int32x8& a)
 { return v_float32x8(_mm256_cvtepi32_ps(a.val)); }
 
+inline v_float32x8 v_cvt_f32(const v_uint32x8& a)
+{ return v_float32x8(_mm256_cvtepi32_ps(a.val)); }
```

Comment:
`_mm256_cvtepi32_ps()` [documentation](https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm256_cvtepi32_ps&expand=1452) has this statement:

> Convert packed **signed** 32-bit integers in a to packed single-precision (32-bit) floating-point elements, and store the results in dst.

1. This doesn't work with unsigned values.
2. Testing doesn't test "unsigned" case.
3. If this works with your code, then you probably don't really need to add this intrinsic.

Reply:
This need to discuss.

Reply:
Removed.